docs/product-updates/2023-09-28-manual-release.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Introducing Manual Release Mode
+tags: [New]
+image: /files/changelog/manual-release.png
+---
+
+![change-comparison.png](/files/changelog/manual-release.png)
+
+Gain greater control over your API Documentation deployments by using the Manual Release Mode for your API documentation.
+
+**Grouping of Changes:** Multiple changes are sometimes part of a single feature or improvement. Manual releases allow these changes to be grouped together under a single changelog entry, rather than being scattered across multiple automatically generated entries.
+
+**Timing:** Not all changes may be immediately relevant to API consumers. The manual release option allows the team to decide when a particular change should be communicated, be it immediately or bundled with other changes at a later date.
+
+**Quality Assurance:** Manual releases allow for an extra layer of QA and collaboration before a change is officially documented.
+
+Learn more in our [Release Management documentation](/help/publish-documentation/release-management/).
+
+### Improvements and fixes
+
+- Change Comparison: Route simplification to enhance UX.
+- CI Deployment settings: Token are now hidden by default.
+- Dashboard: Better deployment warning label.
+- API Documentation: Fix navigation accessibility on darkmode.
```

Comment:
Does multiple servers deserve its own changelog entry, or should we mention it in this list ?

Reply:
I was thinking about adding a proper update for it, but we could favor adding it in this list, wdyt?

Reply:
Added here ✅

Reply:
good call!